backend/Tasque.WebAPI/Controllers/TokenController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace Tasque.Core.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TokenController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+
+        public TokenController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        [AllowAnonymous]
+        [HttpPost]
+        public IActionResult Login([FromBody] object credentials) //Replace <<object>> with UserLoginModel model, for example UserLogin, which has User login and password. Not More
+        {
+            var user = Authenticate(credentials);
+
+            if(user != null)
+            {
+                var token = GenerateToken(user);
+                return Ok(token);
+            }
+            return NotFound("Not found");
+        }
+
+        private string GenerateToken(object user) //Replace object to UserModel (not UserLogin_model)
+        {
+            var jwtOptions = _config.GetSection("JwtIssuerOptions");
+            var securityKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(_config.GetSection("JWTIssuerOptions")["Key"]));
+            var credentials = new SigningCredentials(securityKey, SecurityAlgorithms.HmacSha256);
+
+            var claims = new Claim[] { };//here should be user data, except login and password, such as Name, email ect.
+
+            var token = new JwtSecurityToken(jwtOptions["Issuer"],
+                jwtOptions["Audience"],
+                claims,
+                expires: DateTime.Now.AddDays(1),
+                signingCredentials: credentials);
+
+            return new JwtSecurityTokenHandler().WriteToken(token);
+        }
+
+        private object Authenticate(object credentials)
```

Comment:
btw, if you don't have implementation it's wise to throw NotImplementedException instead of returning null